app/models/map.rb

```diff
@@ -31,27 +31,14 @@ class Map < ActiveRecord::Base
   has_many :annotations, :dependent => :destroy
   belongs_to :user
 
-  has_many :warpables do
-    def public_filenames
-      filenames = {}
-      self.each do |warpable|
-        filenames[warpable.id] = {}
-        sizes = Array.new(Warpable::THUMBNAILS.keys).push(nil)
-        sizes.each do |size|
-          key = size != nil ? size : "original"
-          filenames[warpable.id][key] = warpable.public_filename(size)
-        end
-      end
-      filenames
-    end
-  end
```

Comment:
I've removed this method since it is not being used anywhere else.
Here is the relevent commit https://github.com/publiclab/mapknitter/commit/aadd8b57947f4e204f9a266b9aab4fba66bb79f7
I couldn't find `public_filenames` being called in the current code.